rcl_logging_spdlog/test/fixtures.hpp

```diff
@@ -28,14 +28,15 @@
 #include "rcutils/env.h"
 #include "rcutils/error_handling.h"
 #include "rcutils/process.h"
+#include "rcutils/strdup.h"
 #include "rcutils/types/string_array.h"
 
 #ifdef _WIN32
 #define popen _popen
 #define pclose _pclose
-#define DIR_CMD "dir /B"
+#define DIR_CMD "dir /B /O-D"
```

Comment:
windows needs some help as well, my assumption was incorrect... 😓 

```batch
C:\Users\mrx\work>type nul > 1000

C:\Users\mrx\work>type nul > 1001

C:\Users\mrx\work>dir /B
1000
1001

C:\Users\mrx\work>dir /B /O-D
1001
1000
```

here we need to get the latest log file to compare against the expected strings, adding `/O-D` can make sure listing time descending order.